web-app/messages/en.json

```diff
@@ -665,7 +665,14 @@
           "hiredTitle": "Your Hired Agents",
           "noAgents": "No {type} agents yet",
           "pinnedType": "pinned",
-          "hiredType": "hired"
+          "hiredType": "hired",
+          "Statuses": {
+            "completed": "Completed",
+            "failed": "Failed",
+            "awaitingInput": "Awaiting Input",
+            "awaitingPayment": "Awaiting Payment",
```

Comment:
We changed naming to Payment Processing on purpose, because Awaiting Payment indicates that the user has to do something. Please change to the value we use for the jobs table.